en/simulation/multi_vehicle_simulation_gazebo.md

```diff
@@ -46,6 +49,53 @@ https://youtu.be/aEzFKPMEfjc
 https://youtu.be/lAjjTFFZebI
 {% endyoutube %}
 
+
+
+## Multiple Vehicles with Gazebo based on RTPS {#with_rtps}
+
+To simulate multiple vehicles based on RTPS in Gazebo, use `gazebo_sitl_multiple_run.sh` command in the terminal as described above with `-t px4_sitl_rtps` option (from the root of the *Firmware* tree).  For RTPS, the `micrortps_agent` should be run in the different terminal  for each vehicle.
+
+```
+Tools/gazebo_sitl_multiple_run.sh  -t px4_sitl_rtps [-m <model>] [-n <number_of_vehicles>] [-w <world>] [-s <script>]
+```
+
+> **Note** The model "iris_rtps" is only implemented until now.
+
+
+
+### Build and Test
+
+To build an example setup, follow the step below:
+
+1. Clone the PX4/Firmware code, then build the SITL code:
+
+```bash
```

Comment:
Indent the code block (all lines) with three spaces.

Reply:
Thank you. DONE.